### ISSUE TITLE
Apple Pay updatePlatformSheet doesn't display the errors

### ISSUE THREAD

**Issue by Kuodster** [flutter-stripe/flutter_stripe#1228]
Hello.
Calling updatePlatformSheet for Apple Pay does not display the errors in the sheet.

1. Create a payment sheet with Stripe.instance.createPlatformPayPaymentMethod();
2. Attach a listener to onShippingMethodSelected
3. Try updating the payment sheet with an error:

``` lang-dart
await Stripe.instance.updatePlatformSheet(
        params: PlatformPaySheetUpdateParams.applePay(
          summaryItems: [
            ApplePayCartSummaryItem.immediate(label: 'ERROR', amount: '1.00'),
          ],
          shippingMethods: [],
          errors: [
            ApplePaySheetError.unserviceableShippingAddress(message: 'Some error here'),
          ],
        ),
      );
```

**Expected behavior**
The payment sheet should display an error under the Shipping Address selector.

Currently only shipping options/cart items are updated.

 - Device: iPhone Simulator
 - OS: [e.g. iOS 13, Android 10]
 - Package version: 9.2.0
 - Flutter version 3.7.12

**Comment by Kuodster** [comment]
Created a small PR for this, tested on devices #1230

**Comment by remonh87** [comment]
fixed in stripe_ios 9.2.1
